src/components/AudioTrack/layouts/Row.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="row-track flex flex-row" :class="`size-${size}`">
+    <AudioThumb
+      class="flex-shrink-0"
+      :class="isSmall ? 'w-20 mr-4' : 'w-30 mr-6'"
+      :audio="audio"
+    />
+    <div class="flex" :class="isSmall ? 'flex-row gap-8' : 'flex-col gap-4'">
+      <div class="flex-shrink-0">
+        <p class="font-heading font-semibold text-2xl">{{ audio.title }}</p>
```

Comment:
The Audio title should be a link. I'm not sure if it should be a link _everywhere_, but on the search result page and on related audios for sure.

Reply:
Won't the search results page uses the (upcoming) 'box' layout?

Reply:
I _believe_ the box layout is used in the "all" results view, and the row used when _exclusively_ searching for audio. I agree about linking to the single result view in this Row component.

Reply:
Added the link in c80de9e.